src/OrleansProviders/Streams/Common/Monitors/MonitorAggregationDimensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Orleans.Runtime.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Common
+{
+    public class MonitorAggregationDimensions
+    {
+        /// <summary>
```

Comment:
Every monitor take the related dimension as a input param, so in the monitor, user can define how they want to aggregate their metrics